RELEASE-MANAGEMENT.md

```diff
@@ -2,22 +2,14 @@
 
 This describes the process by which versions of ONNX are officially released to the public.
 
-ISSUE: How do we manage IR and operator version #'s for a given release/pre-release both pre-declaration and post-declaration?
-
-ISSUE: How do we use experimental on operators (or potentially IR features) for a given release/pre-release?
-
-ISSUE: How do we communicate schedules and exit criteria for a given pre-release/release?
-
-ISSUE: How do we stablize one pre-release/release while we are working on subsequent ones?
+Releases
+--------
 
-ISSUE: which files should be included? Probably an archive of all files (minus tests if they get large). How about protoc targets?
+Releases are versioned according to docs/Versioning.md. This describes IR and operator versioning policies, as well as a propose how models themselves should be versioned.
```

Comment:
Can you make "docs/versioning.md" a hyperlink?

"as well as a propose how models" => some words missing or some extra words

Reply:
fixing